tensorboard/webapp/core/store/core_reducers.ts

```diff
@@ -23,6 +23,12 @@ import {CoreState, initialState} from './core_types';
 
 const reducer = createReducer(
   initialState,
+  on(
+    actions.coreConfigLoaded,
+    (state: CoreState, {config}): CoreState => {
+      return {...state, config};
```

Comment:
Given that `Config` represents application level info that is derived from the CLI flags, I think it'd be reasonable to assert that these can never change, or perhaps that they are only ever set once.  e.g. `if (!state.config.equals(config)) throw new Error`